etc/ReplayOfflineConfiguration.py

```diff
@@ -889,7 +895,22 @@
                write_dqm=True,
                write_miniaod=True,
                write_aod=True,
-               dqm_sequences=["@common", "@ecal", "@jetmet", "@hcal", "@L1TEgamma"],
+               dqm_sequences=["@common", "@ecal", "@jetmet", "@L1TMon", "@hcal", "@L1TEgamma"],
+               alca_producers=["TkAlMinBias"],
+               physics_skims=["LogError", "LogErrorMonitor"],
+               scenario=ppScenario)
+
+DATASETS = ["HLTPhysics0", "HLTPhysics1", "HLTPhysics2",
```

Comment:
Not sure where this comes from, but HLTPhysics datasets are duplicated. That's why the replay was not starting, I manually fixed it in the container. The replay is now running, although Jenkins thinks it already failed.